public/css/node-web.scss

```diff
@@ -1,962 +1,234 @@
+      padding-bottom: 80px;
+      border-right: none;
     }
-    span:nth-child(3) {
-      p {
-        padding-top: 18px !important;
-      }
-    }
-  }
-  #reqToFinal-examination {
-    span:nth-child(3) {
-      p {
-        padding-top: 0px !important;
-      }
-    }
-  }
-  #prep-literature {
-    .mandatory {
-      padding-top: 8px !important;
-    }
-    ~ span:nth-child(3) {
-      p {
-        padding-top: 18px !important;
-      }
-    }
-  }
-  #contentAndOutcomes-learningOutcomes {
-    .title-and-info {
-      margin-top: 0;
-      h3 {
-        padding-bottom: 0px;
-      }
-    }
-    .mandatory {
-      padding-top: 8px !important;
-    }
-    span:nth-child(3) {
-      ul {
-        margin-top: 0px;
-      }
-      p {
-        padding-top: 18px !important;
-      }
-    }
-  }
-  #contentAndOutcomes-learningActivities,
-  #contentAndOutcomes-scheduleDetails,
-  #prep-preparations {
-    ~ span:nth-child(3) > p:nth-child(1) {
-      padding-top: 18px !important;
-    }
-  }
 
-  #contacts-teacherAssistants,
-  #contacts-teacher,
-  #contacts-courseCoordinator,
-  #extra-additionalRegulations,
-  #contentAndOutcomes-courseContent,
-  #reqToFinal-otherRequirementsForFinalGrade {
-    .title-and-info {
-      margin-top: 0;
-      h3 {
-        padding-bottom: 0px;
-      }
-    }
-    span:nth-child(3) {
-      p {
-        padding-top: 18px !important;
-      }
-    }
-  }
-  #contacts-examiner {
-    .title-and-info {
-      margin-top: 0;
-      h3 {
-        padding-bottom: 0px;
-      }
-    }
-    span:nth-child(2) {
-      .mandatory {
-        padding-top: 10px !important;
-        padding-bottom: 18px !important;
-      }
-    }
-  }
-  #prep-permanentDisability {
-    .mandatoru {
-      padding-bottom: 0px;
-    }
-    span:nth-child(3) {
-      p {
-        padding-top: 18px !important;
-      }
+    .tab-pane {
+      padding-top: 0;
     }
   }
 
-  p,
-  ul,
-  span {
-    &:last-of-type {
-      margin-bottom: 0;
-    }
-  }
-  ul {
-    margin-top: 1em;
+  .sticky-overview {
+    border: 1px solid var(--color-border);
+    border-top: none;
+    border-bottom: none;
     padding-left: 20px;
-  }
-  span:first-of-type h2 {
-    margin-top: 0;
-  }
-  .title-and-info + p {
-    font-weight: 900;
-  }
-}
-
-.sticky-overview {
-  @extend %bordered-area;
-  border-right-color: $grey;
-  border-top-left-radius: 0;
-  border-top-right-radius: 0;
-  border-bottom: none;
-  padding-left: 0;
-
-  #mainMenu {
-    margin-top: 0;
-  }
+    padding-right: 20px;
+    padding-bottom: 120px;
 
-  @media (min-width: 992px) {
-    #mainMenu.col {
-      max-width: 245px;
-    }
-    .progress-title {
-      display: flex !important;
+    @media (max-width: 991px) {
+      // hacks to disable sticky section menu in smaller screens
+      // TODO: a better solution might be to refactor the sticky feature completly and use position:sticky instead of react-sticky package
+      // hack 1: override position set by react-sticky
```

Comment:
This is part of a bugfix for the sticky feature. My TODO comment are a suggestion for the future, but not really something we need to prioritize now.

`react-sticky` is in maintenance mode has a recommendation to stop using it on its Readme:

_"It is highly recommended that you begin transitioning to another sticky library to ensure better support and sustainability. "_

https://github.com/captivationsoftware/react-sticky